dynawo/sources/Models/Modelica/Dynawo/Examples/IEEE118/BaseClasses/IEEE118Base.mo

```diff
@@ -0,0 +1,1111 @@
+    connect(L_103_104_1.terminal2, B104.terminal);
+    connect(L_103_105_1.terminal1, B103.terminal);
+    connect(L_103_105_1.terminal2, B105.terminal);
+    connect(L_103_110_1.terminal1, B103.terminal);
+    connect(L_103_110_1.terminal2, B110.terminal);
+    connect(L_104_105_1.terminal1, B104.terminal);
+    connect(L_104_105_1.terminal2, B105.terminal);
+    connect(L_105_106_1.terminal1, B105.terminal);
+    connect(L_105_106_1.terminal2, B106.terminal);
+    connect(L_105_107_1.terminal1, B105.terminal);
+    connect(L_105_107_1.terminal2, B107.terminal);
+    connect(L_105_108_1.terminal1, B105.terminal);
+    connect(L_105_108_1.terminal2, B108.terminal);
+    connect(L_106_107_1.terminal1, B106.terminal);
+    connect(L_106_107_1.terminal2, B107.terminal);
+    connect(L_108_109_1.terminal1, B108.terminal);
+    connect(L_108_109_1.terminal2, B109.terminal);
+    connect(L_109_110_1.terminal1, B109.terminal);
+    connect(L_109_110_1.terminal2, B110.terminal);
+    connect(L_110_111_1.terminal1, B110.terminal);
+    connect(L_110_111_1.terminal2, B111.terminal);
+    connect(L_110_112_1.terminal1, B110.terminal);
+    connect(L_110_112_1.terminal2, B112.terminal);
+    connect(L_114_115_1.terminal1, B114.terminal);
+    connect(L_114_115_1.terminal2, B115.terminal);
+    connect(T_8_5_1.terminal1, B8.terminal);
+    connect(T_8_5_1.terminal2, B5.terminal);
+    connect(T_30_17_1.terminal1, B30.terminal);
+    connect(T_30_17_1.terminal2, B17.terminal);
+    connect(T_26_25_1.terminal1, B26.terminal);
+    connect(T_26_25_1.terminal2, B25.terminal);
+    connect(T_38_37_1.terminal1, B38.terminal);
+    connect(T_38_37_1.terminal2, B37.terminal);
+    connect(T_63_59_1.terminal1, B63.terminal);
+    connect(T_63_59_1.terminal2, B59.terminal);
+    connect(T_64_61_1.terminal1, B64.terminal);
+    connect(T_64_61_1.terminal2, B61.terminal);
+    connect(T_65_66_1.terminal1, B65.terminal);
+    connect(T_65_66_1.terminal2, B66.terminal);
+    connect(T_68_69_1.terminal1, B68.terminal);
+    connect(T_68_69_1.terminal2, B69.terminal);
+    connect(T_81_80_1.terminal1, B81.terminal);
+    connect(T_81_80_1.terminal2, B80.terminal);
+    connect(B5_SH1.terminal, B5.terminal);
+    connect(B34_SH1.terminal, B34.terminal);
+    connect(B37_SH1.terminal, B37.terminal);
+    connect(B44_SH1.terminal, B44.terminal);
+    connect(B45_SH1.terminal, B45.terminal);
+    connect(B46_SH1.terminal, B46.terminal);
+    connect(B48_SH1.terminal, B48.terminal);
+    connect(B74_SH1.terminal, B74.terminal);
+    connect(B79_SH1.terminal, B79.terminal);
+    connect(B82_SH1.terminal, B82.terminal);
+    connect(B83_SH1.terminal, B83.terminal);
+    connect(B105_SH1.terminal, B105.terminal);
+    connect(B107_SH1.terminal, B107.terminal);
+    connect(B110_SH1.terminal, B110.terminal);
+
+     //Annotations and calculation parameters
+annotation(preferredView = "text",
+experiment(StartTime = 0, StopTime = 500, Tolerance = 1e-06, Interval = 10),
```

Comment:
Could you remove the experiment line here?

Reply:
done